script/room.js

```diff
@@ -797,6 +797,7 @@ var Room = {
 				$SM.set('stores["'+k+'"]', 0);
 			}
 			
+			k = _(k);
```

Comment:
I will fix it right now.
However, I did the same for outside.js (in that case it's a variable called "name"). I saw that you merged my change, but perhaps I should have used something different there, too, for consistency. I'll give it a try.